# paper-OysterSeed-TimeXTemp

### description of files contained in repository:

#### Methods:  

1. Supplementary_File1:  DDA proteomics analysis workflow that converts files from .raw to .mzXML, searches .mzXML files against C. gigas database, calculates statistics associated with peptide and protein IDs using the Trans Proteomic Pipeline, and correlates protein inferences using ABACUS.   
2. Supplementary_File2: FASTA file containing FASTA sequence file of the C. gigas proteome downloaded from http://gigaton.sigenae.org ('contigs.fasta.transdecoder.pep.gz') and common contaminants downloaded from the crapOME (Mellacheruvu et al. Nat Methods 2013).
2. Supplementary_File3:  COMET 2016.01 'comet.params.high-low' search parameters used in COMET analysis of mass spectrometry data download from http://comet-ms.sourceforge.net/parameters/parameters_201601/comet.params.high-low
3. Supplementary_File4:  ABACUS parameter file used in ABACUS analysis of mass spectrometry data

#### Data:  

1. Supplementary_Data1:  
2. Supplementary_Data2:  ABACUS_output.tsv.gz file output from ABACUS analysis
3. Supplementary_Data3:  Average NSAF values for all proteins (listed in columns) detected in all samples (listed in rows) where NSAF values of 0 have been converted to 0.1.
4. Supplementary_Data4:  Proteins and their clusters assigned by hierarchical clustering analysis
5. Supplementary_Data5:  ASCA analysis summary
6. Supplementary_Data6:  ASCA-generated temperature effect PCA PC1 loadings values for all proteins detected across silos
7. Supplementary_Data7:  Log base 2 fold change and Chi squared raw and corrected p-values of all proteins detected across silos
8. Supplementary_Data8:  
9. Supplementary_Data9:  Node attribute file for proteins containing unique names for proteins (Uniprot entry gene name with protein name suffix appended to distinguish proteoforms), log fold change values, and FDR-corrected Chi squared proportions test p-values
10. Supplementary_Data10:
11. Supplementary_Data11:

